File: Feitos em C/1238.c
#include <stdio.h>
int main() {
	int times, i, j;
	scanf("%d",&times);
	for(i=0;i<times;i++){
		char word1[50],word2[50];
		for(j=0;j<50;j++){
			word1[j]=48;
			word2[j]=48;
		}
		scanf("%s %s",&word1,&word2);
		for(j=0;j<50;j++){
			if(word1[j]==0){
				word1[j]=48;
			}
			if(word2[j]==0){
				word2[j]=48;
			}
		}
		for(j=0;j<50;j++){
			if(word1[j]!=48){
				printf("%c",word1[j]);
			}
			if(word2[j]!=48){
				printf("%c",word2[j]);
			}
		}
		printf("\n");
	}
    return 0;
}

File: Feitos em C Sharp/1238.cs
using System;
namespace ConsoleApp4 {
    class Program {
        static void Main(string[] args) {
			int times = int.Parse(Console.ReadLine());
			for (int i = 0; i < times; i++) {
				string[] phrase = Console.ReadLine().Split();
				string word1 = phrase[0];
				string word2 = phrase[1];
				int num1 = word1.Length, aux1 = 0;
				int num2 = word2.Length, aux2 = 0;
				for (int j = 0; j < 50; j++) {
					if (aux1 < num1) {
						Console.Write(word1.Substring(j, 1));
						aux1++;
					}
					if (aux2 < num2) {
						Console.Write(word2.Substring(j, 1));
						aux2++;
					}
				}
				Console.WriteLine();
			}
        }
    }
}
